examples/distributed-key-value-store/README.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+This example showcases a basic distributed key-value store implemented using **libp2p**, along with the mDNS and Kademlia protocols.
+
+## Usage
+
+### Key-Value Store
+
+1.  Open two terminal windows.
+
+2.  In terminal one, type `PUT my-key my-value` and press Enter.
```

Comment:
Sorry to be nit-picky but we need a `cargo run` first, don't we?

Reply:
@thomaseizinger No problem, this is normal, the work has to be done properly :) and you're right, I've just modified that by adding the `cargo run` command in the first step and pushed again.